dev-libs/pigpio/pigpio-66.ebuild

```diff
@@ -27,31 +27,13 @@ src_prepare() {
 
 src_compile() {
 	emake
-	if use python && distutils-r1_src_compile ; then
-		distutils-r1_src_compile
-	fi
+	use python && distutils-r1_src_compile
 }
 
 src_install() {
-	dolib.so libpigpio.so.0
-	dolib.so libpigpiod_if.so.0
-	dolib.so libpigpiod_if2.so.0
-	dosym libpigpio.so.0 /usr/$(get_libdir)/libpigpio.so
-	dosym libpigpiod_if.so.0 /usr/$(get_libdir)/libpigpiod_if.so
-	dosym libpigpiod_if2.so.0 /usr/$(get_libdir)/libpigpiod_if2.so
-	insinto /usr/include/
-	doins pigpio.h
-	doins pigpiod_if.h
-	doins pigpiod_if2.h
-	dobin pigpiod
-	dobin pig2vcd
-	dobin pigs
-	doman *.1
-	doman *.3
+	emake DESTDIR="${D}" install
```

Comment:
it can really be reduced to just `default`. See here fir more info:
https://devmanual.gentoo.org/ebuild-writing/eapi/index.html

Was introduced in `EAPI=4`

Reply:
Well, i still need to install the init (openrc and system) files and need to have the conditional python install, thus the modification of the src_install() phase.
In that case, I'm not sure i can just omit the "emake DESTDIR="${D}" install" ? Or is there some way to amend (instead of replace the src_install() function?

Reply:
@github-k8n, `default` does not prevent you from installing the files you need, you still can do so after `default` call